src/views/UserLocation.vue

```diff
@@ -18,8 +18,7 @@
                 </form>
                 <!-- MapView component to display the map -->
                 <MapView ref="mapView" />
-                <!-- Button to save location to the current trip -->
-                <button @click="saveLocationToTrip" class="btn btn-success mt-3">Save Location to Current Trip</button>
+                <button @click="clearLocations" class="btn btn-danger mt-3">Clear Saved Locations</button>
```

Comment:
We should use the component here, I'll add it real quick after this goes out though